working-groups/community_meeting/26-01-2023.md

```diff
@@ -0,0 +1,47 @@
+# 26-01-2023 [community]  
+## Agenda - Open to Discussion
+- GSoC & GSoD participation
+- Publications about moja global projects
+- Participation in conferences/meetups
+- Implementation of metrics to track progress
+
+## Discussion
+
+- Creating Blogs for the moja global organization. Like the technology on which we are working like Vuejs for frontend and all.
+- Example - Harsh is taking the blog on “DevOps practises” and “FLINT.Cloud GCP” by Gopi 
+- You being recognized by the name and able to learn lot’s of thing regarding the MojaGlobal just by writing about the same.
+- Platform to write Blogs: Medium 
+
+##Reach out initiative :
+- College Outreach - Talks in college and collaborative work done by students (resources will be provided)
+- Conferences can be attended to write blog posts on Medium
+- Increase the reach of Moja Global
+- Communications team along with Harsh is processing the idea for creating online reach via Social Media.
+
+## Tips to keep in mind: 
+- Dont become an organiser, be the person to promote Moja Global
```

Comment:
Great work @meenaljain0303 🚀 
I have a minor change to suggest here: 
- Can you please correct from `Dont` => `Don't`?